webapp/layouts/default.vue

```diff
@@ -200,23 +240,24 @@ export default {
       }
       return this.$route.path.indexOf(url) === 0
     },
-    unfolded: function() {
-      document.getElementById('nav-search-box').classList.add('unfolded')
+    toggleMobileMenuView() {
+      this.toggleMobileMenu = !this.toggleMobileMenu
+    },
+    redirectToRoot() {
+      this.$router.replace('/')
+      this.fetchPosts({ i18n: this.$i18n, filter: {} })
     },
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-07-10T22:14:59Z" title="Thursday, July 11th 2019, 12:14:59 am +02:00">Jul 11, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -206,6 +245,21 @@ export default {
     foldedup: function() {
       document.getElementById('nav-search-box').classList.remove('unfolded')
     },
+    toggleMobileMenuView() {
+      this.toggleMobileMenu = !this.toggleMobileMenu
+    },
+    redirectToRoot() {
+      this.$router.replace('/')
+      this.fetchPosts({ i18n: this.$i18n, filter: {} })
+    },
```


this certainly feels like less than an ideal situation, and I'm open to other ideas how to clear the search queries for `Hashtags` and also refetch all the `Posts` to clear the search for `Hashtags/Categories`
@Tirokk @roschaefer @ogerly @ulfgebhardt 

it works with `location.reload()` as well, but I think that's worse.
it is not great user experience because it's not clear that anything is happening... maybe there could be a spinner/loader, but it's challenging because the logo link lives in `layouts/default.vue`, whereas the `Posts` list lives in `webapp/pages/index.vue`